src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -2532,6 +2532,7 @@ public static partial class Math
     {
         public const double E = 2.718281828459045;
         public const double PI = 3.141592653589793;
+        public const double Tau = 6.283185307179586;
```

Comment:
Just noting for anyone that comes looking, the value we put in the `src` is the actual value of PI to at least 17 significant digits + 1 rounding digit (or to at least 9 significant digits + 1 rounding digit for `float`).

The value here is the actual `double` or `float` value (which may be different due to rounding as the actual value may not be exactly representable) formatted as a string with at least enough precision to correctly roundtrip. Hence why the `ref float` versions differ slightly from the "true digits" of PI or TAU

```
REF FLT: 3.1415927
SRC FLT: 3.1415926 5
REF DBL: 3.1415926 53589793
SRC DBL: 3.1415926 53589793 23846
TRU PI:  3.1415926 53589793 23846 2643383279502884197169399375105820974...
```

```
REF FLT: 6.2831855
SRC FLT: 6.2831853 07
REF DBL: 6.2831853 07179586
SRC DBL: 6.2831853 07179586 476925
TRU TAU: 6.2831853 07179586 476925 286766559005768394338798750211641949...
```